core/protobuf/src/proto/journal/file.proto

```diff
@@ -33,11 +33,12 @@ message CompleteFileEntry {
   optional int64 op_time_ms = 4;
 }
 
-// next available id: 4
+// next available id: 5
 message DeleteFileEntry {
   optional int64 id = 1;
   optional bool recursive = 2;
-  optional int64 op_time_ms = 3;
+  optional bool alluxioOnly =3;
```

Comment:
we should not change the tag index. namely, still keep 
`optional int64 op_time_ms = 3` and add `optional bool alluxioOnly =4;` to keep backward compatibility